lib/node/nodes/trade-area/trade-area.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
```

Comment:
I prefer to keep the nodes inside `lib/node/nodes` and not create new directories.